### ISSUE TITLE
Stop sending cookies in mobile web

### ISSUE THREAD

**Issue by manumoreira** [instedd/surveda#2241]
We are currently using cookies to guarantee that the respondent who's answering the survey is the same one that received the SMS and opened the link the first time.
In the last implementations, this feature is adding more problems than benefits. The typical error is Unauthorized. 
[Here's a convo about this](https://manas.slack.com/archives/C3KKQKHLG/p1678292716750619?thread_ts=1677782971.998559&cid=C3KKQKHLG)
We will not lose much if we stop using cookies,  worst case the user will share the link and someone else will answer the same survey but it will only be one answered survey in the end.

**Comment by ysbaddaden** [comment]
@manumoreira Verified on my local. I created a MobileWeb questionnaire, and started survey. I followed the URL from one of the queued message in Nuntium to start a Survey in a private browser window. No cookies were created during the survey.

I could start the survey on different windows (which was a drawback) but once the survey was completed in a window, the survey wouldn't start anymore (survey already taken).

AFAIK this is working just as expected.

**Comment by manumoreira** [comment]
@ysbaddaden have you checked completing the survey in parts? Does it track the progress correctly? Say you start the survey in browser A, get to step 3, and a while after, start the survey in browser B and start from step 4?

**Comment by ysbaddaden** [comment]
@manumoreira they interfere with each other. 

**1st attempt:**

I opened the same survey in two different windows. I got the intro page and continued on to the first question on both windows A and B. I answered the first and second question in window A, I then answered the first question in window B and it led me directly to the third question.

If I reload a window, it opens the intro page again, but when I take the survey I'm immediately pushed to the third question.

I answered the last question on window A, when I returned to window B, I couldn't even enter a response and the "survey already taken" message gets displayed.

**2nd attempt**

I loaded another respondent in two windows again, and went from intro to the first question on both. I answered the first two questions in window A, then tried to answer the first question in window B... which actually answered the third question. I didn't notice, but the window updated itself to reflect the current state of the respondent from the backend :ok_hand:
